.chloggen/opampextension-add-os-desc-semconv.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: opampextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Report OS description semantic convention as a part of non-identifying agent description.
```

```suggestion
note: Report OS description semantic convention (`os.description`) as a part of non-identifying agent description.
```
Not blocking, but I think it would be helpful for users to know which attribute is being reported without needing to look it up.